src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/net/NetServer.java

```diff
@@ -107,6 +111,7 @@ public void run() {
     try {
       server.start();
       LOG.info("gRPC server started successfully!");
+      running.set(true);
```

Comment:
Can I suggest an alternative ? Can we add a prehook and post hook which will be empty and then for test we override these methods to set the variable and we can test it ? I assume this variable is only for test purposes only ?

Reply:
Done